fastn-core/src/library2022/processor/google_sheets.rs

```diff
@@ -0,0 +1,98 @@
+const GOOGLE_SHEET_API_BASE_URL: &str = "https://docs.google.com/a/google.com/spreadsheets/d";
+
+static GOOGLE_SHEETS_ID_REGEX: once_cell::sync::Lazy<regex::Regex> =
+    once_cell::sync::Lazy::new(|| regex::Regex::new(r"/spreadsheets/d/([a-zA-Z0-9-_]+)").unwrap());
+
+pub fn extract_google_sheets_id(url: &str) -> Option<String> {
```

Comment:
```suggestion
pub(crate) fn extract_google_sheets_id(url: &str) -> Option<String> {
```

Reply:
Convert the functions that are not going to used outside crate as `pub(crate)`

Reply:
Got it, thanks